src/pipecat/processors/aggregators/llm_response.py

```diff
@@ -243,6 +260,20 @@ def __init__(self, *, context: OpenAILLMContext, **kwargs):
 
         self._context = context
         super().__init__(**kwargs)
+        # TODO-CB: thanks, I hate it
+        self._messages = context.messages
+    
+        
+    async def _set_tools(self, tools: List):
```

Comment:
`set_messages()` is public why not `_set_tools()`?

Reply:
Fixed 👍